zilliqa/src/consensus.rs

```diff
@@ -111,6 +111,13 @@ type EarlyProposal = (
     EthTrie<MemoryDB>,
 );
 
+#[derive(Debug)]
+struct NewViewMessageStore {
+    peer_id: PeerId,
+    new_view: NewView,
+    resend_count: u8,
```

Comment:
What is this count used for? Do you care that it will overflow quite quickly (5 seconds * 255 = ~21 minutes)?

Reply:
doh, should have been u16. - removed now anyway.